apps/index.html

```diff
@@ -305,6 +305,22 @@
       </div>
 
```

Comment:
Hi @mquandalle and thanks for submitting this! I have some small comments that might improve this pull request. I'll let Kenton be the one to merge it or not, but wanted to give you this feedback with the goal of hopefully having you make a few small changes to improve the submission.

Even without the changes, I think this is shippable, but having noticed some things, I thought it'd be nice to share them.